### PR TITLE
[12.0][MIG] rma_repair

### PR DESCRIPTION
**Pull request by grindtildeath**
Depends on : 
 - [ ] https://github.com/OCA/manufacture/pull/342

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Ping @lreficent

File: rma_repair/models/rma_order.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class RmaOrder(models.Model):
+    _inherit = "rma.order"
+
+    @api.multi
+    def _compute_repair_count(self):
+        for rma in self:
+            repairs = rma.mapped('rma_line_ids.repair_ids')
+            rma.rma_count = len(repairs)
```
Comment (bjeficent) [inline rma_repair/models/rma_order.py]: ```rma_count```? Should not be ```repair_count```?

Comment (AaronHForgeFlow) [reply rma_repair/models/rma_order.py]: I just created a PR for that https://github.com/grindtildeath/stock-rma/pull/1/

**Comment by AaronHForgeFlow** [comment]
Thank you @grindtildeath  OCA/manufacture#342 is merged, so we can know remove the oca_dependencies and move forward.

Regards

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**